docs: getting started installation and next lint removal (#84781)
vercel/next.js: docs/01-app/01-getting-started/01-installation.mdx

If you use ESLint, create an explicit config (recommended `eslint.config.mjs`). ESLint supports both [the legacy `.eslintrc.*` and the newer `eslint.config.mjs` formats](https://eslint.org/docs/latest/use/configure/configuration-files#configuring-eslint). See the [ESLint API reference](/docs/app/api-reference/config/eslint#with-core-web-vitals) for a recommended setup.

> > **Good to know**: Starting with Next.js 16, linting no longer runs automatically during `next build`. You now control when linting runs by invoking your linter through npm scripts.

Reviewer: ```suggestion
> **Good to know**: Starting with Next.js 16, `next build` no longer runs the linter automatically. Instead, you can run your linter through NPM scripts.
```

Reviewer: I felt this was more natural, but feel free to abandon
